content/_includes/FusionPools.md

```diff
@@ -16,29 +16,29 @@ Using a special vdev drastically speeds up random I/O and cuts the average spinn
 
 Go to **Storage > Pools**, click **ADD**, and select **Create new pool**.
 
-A pool must always have one normal (non-dedup/special) vdev before other devices can be assigned to the special class.
+A pool must always have one normal (non-dedup/special) VDEV before other devices can be assigned to the special class.
 Configure the **Data VDevs**, then click **ADD VDEV** and select **Metadata**.
 
 ![CreateMetadataVdev](/images/CORE/13.0/CreateMetadataVdev.png "Create Metadata VDev Screen")
 
 Add SSDs to the new **Metadata VDev** and select the same layout as the **Data VDevs**.
 
 {{< hint warning >}}
-The metadata special vdev is critical for pool operation and data integrity, so you must protect it with hot spare(s).
+The metadata special VDEV is critical for pool operation and data integrity, so you must protect it with hot spare(s).
 {{< /hint >}}
 
 {{< expand "UPS Recommendation" "v" >}}
 When using SSDs with an internal cache, add uninterruptible power supply (UPS) to the system to help minimize the risk from power loss.
 {{< /expand >}}
 
-Using special vdevs identical to the data vdevs (so they can use the same hot spares) is recommended, but for performance reasons you can make a different type of vdev (like a mirror of SSDs).
-In that case you must provide hot spare(s) for that drive type as well. Otherwise, if the special vdev fails and there is no redundancy, the pool becomes corrupted and prevents access to stored data.
+Using special VDEVs identical to the data VDEVs (so they can use the same hot spares) is recommended, but for performance reasons you can make a different type of VDEV (like a mirror of SSDs).
+In that case you must provide hot spare(s) for that drive type as well. Otherwise, if the special VDEV fails and there is no redundancy, the pool becomes corrupted and prevents access to stored data.
```

Comment:
```suggestion
We recommend using special VDEVs identical to the data VDEVs (so they can use the same hot spares), but for performance reasons, you can make a different type of VDEV (like a mirror of SSDs).
In that case, you must provide hot spare(s) for that drive type as well. Otherwise, if the special VDEV fails and there is no redundancy, the pool becomes corrupted and prevents access to stored data.
```